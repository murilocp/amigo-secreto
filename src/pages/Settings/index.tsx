import React from "react";
import { Card, Footer, Form, ParticipantList } from "../../components";
import { Container } from "./styles";

const Settings: React.FC = () => {
  return (
    <Container>
      <Card>
        <section>
          <h2>Vamos come√ßar!</h2>
          <Form />
          <ParticipantList />
          <Footer />
        </section>
      </Card>
    </Container>
  );
};

export default Settings;
