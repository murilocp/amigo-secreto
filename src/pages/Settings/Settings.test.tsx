import React from "react";
import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Settings from ".";

const mockUseNavigate = jest.fn();

jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockUseNavigate,
  };
});

describe("Settings page", () => {
  test("Should render", () => {
    const { container } = render(
      <RecoilRoot>
        <Settings />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});
