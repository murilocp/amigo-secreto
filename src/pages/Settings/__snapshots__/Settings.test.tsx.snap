// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings page Should render 1`] = `
<div>
  <div
    class="sc-eCYdqJ"
  >
    <div
      class="sc-bczRLJ jvcWzg"
    >
      <section>
        <h2>
          Vamos come√ßar!
        </h2>
        <form
          class="sc-dkzDqf hWOVKR"
        >
          <div
            class="grupo-input-btn"
          >
            <input
              placeholder="Insira os nomes dos participantes"
              type="text"
              value=""
            />
            <button
              disabled=""
            >
              Adicionar
            </button>
          </div>
          
        </form>
        <ul />
        <footer
          class="sc-gsnTZi iFbUYD"
        >
          <button
            disabled=""
          >
            Iniciar brincadeira!
          </button>
          <img
            alt="Sacolas de compras"
            src="/imagens/sacolas.png"
          />
        </footer>
      </section>
    </div>
  </div>
</div>
`;
